src/SFA.DAS.EmployerFinance.Web/Authentication/ServiceCollectionOidcExtensions.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerFinance.Configuration;
+using SFA.DAS.EmployerFinance.Web.Cookies;
+
+namespace SFA.DAS.EmployerFinance.Web.Authentication
+{
+    public static class ServiceCollectionOidcExtensions
+    {
+        public static IServiceCollection AddAndConfigureAuthentication(this IServiceCollection services, IOidcConfiguration oidcConfig, bool isDevelopment)
```

Comment:
What do you think to renaming this to `AddOidcAuthentication`?

Reply:
Happy to - done.